contents/tutorials/zendesk-reports.md

```diff
@@ -0,0 +1,167 @@
+---
+title: How to set up Zendesk reports
+date: 2024-06-28
+author:
+ - ian-vanagas
+tags:
+ - data warehouse
+---
+
+Creating and analyzing reports of Zendesk data along with product data helps you understand support performance, identify problem areas, and provide a better customer experience.
+
+In this tutorial, we show how you can create Zendesk reports in PostHog by connecting it as a data source and then creating insights like the number of tickets.
```

Comment:
Sentence is a bit long too. "Reports" kind of implies insights, so we can remove it
```suggestion
In this tutorial, we show how you can create Zendesk reports in PostHog by connecting it as a data source. 
```